tgui/packages/tgui/interfaces/MarkdownViewer.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file MarkdownViewer.tsx
+ * @author ZephyrTFA
+ * @license MIT
+ */
+
+import { Component } from 'inferno';
+import { marked } from 'marked';
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+type MarkdownViewerData = {
+  title: string;
+  content: string;
+  author: string;
+};
+
+export const MarkdownViewer = (props: any, context: any) => {
+  let { data } = useBackend<MarkdownViewerData>(context);
+  return (
+    <MarkdownViewerImpl
+      title={data.title || 'No Title'}
+      content={data.content || 'No Content'}
+      author={data.author || 'Unknown Author'}
+    />
+  );
+};
+
+export class MarkdownViewerImpl extends Component<MarkdownViewerData> {
+  render() {
+    return (
+      <Window title={this.props.title}>
+        <Window.Content>
```

Comment:
I did some testing, and doing it like this will allow it to look just like white paper does.
```suggestion
      <Window title={this.props.title} theme="paper">
        <Window.Content backgroundColor={'#FFFFFF'}>
```

![image](https://user-images.githubusercontent.com/58045821/233735333-8c17854a-baf5-479d-b379-851dd5630dff.png)